pkg/driver/bacnet/merge/air_temperature.go

```diff
@@ -88,11 +90,37 @@ func (t *airTemperature) GetAirTemperature(ctx context.Context, request *traits.
 	return t.ModelServer.GetAirTemperature(ctx, request)
 }
 
+func (t *airTemperature) isArtusUnit() bool {
+	if (t.config.Metadata != nil) && (t.config.Metadata.Product != nil) {
+		if t.config.Metadata.Product.Model == "artus" {
+			return true
+		}
+	}
+	return false
+}
+
 func (t *airTemperature) UpdateAirTemperature(ctx context.Context, request *traits.UpdateAirTemperatureRequest) (*traits.AirTemperature, error) {
 	if request.GetState().GetTemperatureSetPoint() == nil {
 		return t.GetAirTemperature(ctx, &traits.GetAirTemperatureRequest{Name: request.Name})
 	}
 	newSetPoint := float32(request.GetState().GetTemperatureSetPoint().GetValueCelsius())
+
+	if t.isArtusUnit() {
```

Comment:
I'd be happy if this condition were more like `if t.config.SetPointLow != nil && t.config.SetPointHigh != nil` instead of using the metadata to change logic. Can't think of any reason why not to do it this way